.tekton/pipeline-service-test.yaml

```diff
@@ -48,61 +50,25 @@ spec:
                 echo -n "$CLUSTER_NAME" | tee $(results.cluster-name.path)
       - name: clean-clusters
         runAfter:
-          - "produce-cluster-name"
-        params:
-          - name: namespace
-            value: "$(params.namespace)"
-          - name: cluster-name
-            value: "$(tasks.produce-cluster-name.results.cluster-name)"
+          - "fetch-repository"
```

Comment:
Shouldn't this ideally run after everything is done as the last step? And not after fetch-repository?